README.md

```diff
@@ -1,23 +1,44 @@
-# purescript-http-methods
+# Http Methods
 
-[![Latest release](http://img.shields.io/github/release/purescript-contrib/purescript-http-methods.svg)](https://github.com/purescript-contrib/purescript-http-methods/releases)
-[![Build status](https://travis-ci.org/purescript-contrib/purescript-http-methods.svg?branch=master)](https://travis-ci.org/purescript-contrib/purescript-http-methods)
-[![Pursuit](http://pursuit.purescript.org/packages/purescript-http-methods/badge)](http://pursuit.purescript.org/packages/purescript-http-methods/)
-[![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-lightgrey.svg)](http://github.com/garyb)
-[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-lightgrey.svg)](http://github.com/thomashoneyman)
+[![CI](https://github.com/purescript-contrib/purescript-http-methods/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-http-methods/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](https://img.shields.io/github/release/purescript-contrib/purescript-http-methods.svg)](https://github.com/purescript-contrib/purescript-http-methods/releases)
+[![Pursuit](https://pursuit.purescript.org/packages/purescript-http-methods/badge)](https://pursuit.purescript.org/packages/purescript-http-methods)
+[![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-teal.svg)](https://github.com/garyb)
+[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-teal.svg)](https://github.com/thomashoneyman)
 
 HTTP method type.
 
 ## Installation
 
+Install `http-methods` with [Spago](https://github.com/purescript/spago):
+
+```sh
+spago install http-methods
 ```
-bower install purescript-http-methods
-```
+
+## Quick start
+
+The quick start hasn't been written yet (contributions are welcome!). The quick start covers a common, minimal use case for the library, whereas longer examples and tutorials are kept in the [docs directory](./docs.)
 
 ## Documentation
 
-Module documentation is [published on Pursuit](http://pursuit.purescript.org/packages/purescript-http-methods).
+`http-methods` documentation is stored in a few places:
+
+1. Module documentation is [published on Pursuit](https://pursuit.purescript.org/packages/purescript-http-methods).
+2. Written documentation and [the changelog](./docs/CHANGELOG.md) are kept in [the docs directory](./docs).
```

Comment:
If there's not much reason for a documentation directory we could perhaps just remove them and treat this as a special case. Perhaps we could add to the library summary the RFCs this is based on (HTTP/1.1 with RFC 2518 and RFC 5789)?

Reply:
I'm up for removing the `docs` repo, but we would need to move the `CHANGELOG.md` file into the root folder.

Reply:
Looks like we do need to move the CHANGELOG and the CONTRIBUTING files into the repository root. When doing that, we could remove this line altogether (because there won't be a docs directory anymore). Remember that we'll also need to fix the CONTRIBUTING link later in the README file.

Reply:
I think leaving `CONTRIBUTING.MD` in the `.github` folder [is a valid choice](https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file) based on GH's help page. Whether we want to leave it there or move it to the root folder is a separate matter.

I think moving it to the root folder is better as it increases visibility of it.